src/config/konnectors.json

```diff
@@ -74,7 +77,14 @@
     "category":"telecom",
     "color":{"hex":"#FF6122","css":"#FF6122"},
     "dataType":["bill", "geopoint", "phonecommunicationlog"],
-    "fields":{"access_token":{"type":"hidden"},"folderPath":{"type":"folder","advanced":true}},
+    "fields":{"access_token":{"type":"hidden"},"folderPath":{"type":"folder","advanced":true}, "agreement": {"type": "checkbox", "description": true}},
+    "additionnalSuccessMessage": {
+      "message": "connector.orange.message.delay"
+    },
+    "description": {
```

Comment:
I feel that this attributes `description` is too confusing, has it contains only boolean attributes instead of an expected string description.
I suggest to replace it by something like `descriptions` or `hasDescriptions`.